core/server/helpers/t.js

```diff
@@ -0,0 +1,31 @@
+// # t helper
+// i18n: Translatable handlebars expressions for templates of the front-end and themes.
+// Front-end: .hbs templates in core/server, overridden by copies in themes. Themes: in content/themes.
+//
+// Usage examples, for example in .hbs theme templates:
+// {{t "Get the latest posts delivered right to your inbox"}}
+// {{{t "Proudly published with {ghostlink}" ghostlink="<a href=\"https://ghost.org\">Ghost</a>"}}}
+//
+// To preserve HTML, use {{{t}}}. This helper doesn't use a SafeString object which would prevent escaping,
+// because often other helpers need that (t) returns a string to be able to work as subexpression; e.g.:
+// {{tags prefix=(t " on ")}}
+
+var proxy = require('./proxy'),
+    jp = require('jsonpath'),
+    i18n = proxy.i18n,
+    settingsCache = proxy.settingsCache,
+    bindings = {},
+    path;
+
+module.exports = function t(text, options) {
+    var prop, theme;
+    for (prop in options.hash) {
+        if (options.hash.hasOwnProperty(prop)) {
+            bindings[prop] = options.hash[prop];
+        }
+    }
+    bindings.defaultString = text;
+    theme = settingsCache.get('active_theme') || 'casper';
```

Comment:
Why falling back to `caper` here`? `active_theme` should be never null.

Reply:
Also unneeded now, probably a just in case from some very old test that didn't load a theme. This has been also removed because, with today's commit for separate objects, there is no more need for the theme prefix and the name of the theme in the JSON translation files.